core/app/models/spree/option_value.rb

```diff
@@ -16,6 +16,7 @@ class OptionValue < Spree::Base
     scope :filterable, lambda {
       joins(:option_type).
         where(OptionType.table_name => { filterable: true }).
+        order(position: :asc).
```

Comment:
@aleksandarpetrushev 

1. this order is ambiguous as both option values and option types have the `position` attribute
2. Also let's not modify this scope, let's add the order in the service - https://github.com/spree/spree/blob/a0137d7aa25fa48639c88e5baef11bd236559812/core/app/finders/spree/option_values/find_available.rb

Reply:
3. We need tests to back it up